src/components/NavigasjonsTopp.tsx

```diff
@@ -11,24 +12,53 @@ interface NavigasjonsToppProps {
   lenker: Lenke[];
 }
 
+const NavigasjonLenkeListe = styled.ul`
+  padding: 0;
+  margin-bottom: 2.5em;
+`;
+
+const NavigasjonsLenke = styled.li`
+  display: inline-block;
+  padding-top: 0.25em;
+  padding-bottom: 0.5em;
+  padding-right: 0.5em;
+  margin: 0 2em 0 0;
+
+  text-transform: uppercase;
+
+  a {
+    font-weight: bold;
+    text-decoration: none;
+    color: black;
+    font-size: 1.125em;
+  }
+`;
+
+const StyledLink = styled(Link)<{ aktiv: number }>`
+  align-items: center;
+  cursor: pointer;
+
+  ${(props) =>
+    props.aktiv &&
+    css`
+      border-bottom: 4px var(--a-blue-500) solid;
+    `}
+  &:hover {
+    border-bottom: 4px var(--a-blue-500) solid;
+  }
+`;
+
 const NavigasjonsTopp = ({ lenker }: NavigasjonsToppProps): ReactElement => (
-  <header className="navigasjon">
-    <ul role="navigation" className="navigasjonsLenkeListe">
-      {lenker.map((lenke) => {
-        return (
-          <li className="navigasjon__element" key={lenke.url}>
-            <Link
-              className={`navigasjon__element__inner${
-                lenke.aktiv ? "--active" : ""
-              }`}
-              to={lenke.url}
-            >
-              {lenke.tittel}
-            </Link>
-          </li>
-        );
-      })}
-    </ul>
+  <header>
+    <NavigasjonLenkeListe>
+      {lenker.map((lenke) => (
+        <NavigasjonsLenke key={lenke.url}>
+          <StyledLink aktiv={lenke.aktiv ? 1 : 0} to={lenke.url}>
```

Comment:
Er ikke `aktiv` allerede en bool, hvorfor må man sende `1` eller `0`?

Reply:
Fikk en snål warning ved bruk av bool her så måtte hacke rundt den :confused: :
`Warning: Received `false` for a non-boolean attribute `aktiv`.
If you want to write it to the DOM, pass a string instead: aktiv="false" or aktiv={value.toString()}.
If you used to conditionally omit it with aktiv={condition && value}, pass aktiv={condition ? value : undefined} instead.`

Reply:
Aha! En temporary fix fra 2017 forble temporary virker det som? 
https://github.com/styled-components/styled-components/issues/1198#issuecomment-336621217